### DISCUSSION TITLE
DiePi-Software | CamillaDSP

### DISCUSSION THREAD

**Discussion by DeLub** [MichaIng/DietPi#7983] [Software Request]
#### Formal software information
- Software title | CamillaDSP
- Short description | A IIR and FIR engine for equalisation, crossovers, room correction etc.
- Official URL (if available) | https://henquist.github.io
- GitHub URL (if available) | https://github.com/HEnquist/camilladsp
- (Official) install documentation | https://henquist.github.io/0.6.0/

#### Are there similar/alternative software titles available with DietPi-Software?
- ffmpeg

#### What makes your requested software better than the above solutions, if available?
- lightweight
- more flexible
- easier configuration, esp. via gui
- easier to run as a service

#### How can DietPi make the installation easier or compatible, than following the install instructions or do APT installation, if available?
- install dependencies
- create default configuration
- install service
- install gui + dependencies

#### Can you provide the installation steps that you would suggest DietPi-Software to do?

1. Install ALSA
2. Install CamillaDSP
3. Install Python
4. Install dependencies
5. Install CamillaDSP GUI

#### Are you willing to help maintaining the software installation, e.g. in case of needed setup changes due to updates etc.? This is not needed, but could speed up our decision to implement it, as man power is always a topic :wink:.
- Yes

**Comment by alkmarmasor** [comment]
Up - very usefull for usb\wifi dsp
